### ISSUE TITLE
Cannot keyboard focus/interact legend paging buttons

### ISSUE THREAD

**Issue by msdci** [highcharts/highcharts#7415]
#### Expected behaviour
Should be able to navigate with keyboard to focus/interact with legend paging buttons

#### Actual behavior
Cannot focus/interact legend paging buttons with keyboard, thus users cannot page through legends while navigating with keyboard only

#### Affected browser(s)
Chrome, Edge, IE

**Comment by pawelfus** [comment]
Hi @msdci 

Thank you for reporting this issue.

Demo: http://jsfiddle.net/5ze21cvy/

FYI @oysteinmoseng

**Comment by oysteinmoseng** [comment]
Internal note: Legend works through all elements, even hidden ones, but the list is not scrolled as we move down. We should probably automatically scroll the list as we progress through.

**Comment by oysteinmoseng** [comment]
Legend now scrolls automatically as you navigate through items: http://jsfiddle.net/oysteinmoseng/3hwpx4nb/show
